### ISSUE TITLE
Improve website design

### ISSUE THREAD

**Issue by alarsyo** [s-society/sharpboy#9]
At least some custom CSS instead of the default theme.

Some ideas I could use : 

- [Blank Theme](https://github.com/garrettboatman/Blank-Theme-Jekyll)

**Comment by bandang0** [comment]
Hey guys I just wrote a Jekyll site [here](http://theia.hopto.org:56000) for the [Theia Project](https://github.com/bandang0/theia) that looks **pretty sleek** if you want some help on this one.

**Comment by alarsyo** [comment]
Hi, to be fair I don't see any difference between your site and ours, it looks like we're using the exact same base theme.

**Comment by bandang0** [comment]
Hi,
Fortunately it's not only about the base theme! It's also about having some content other than the list of posts on the front page, and putting in releases without having to link to the github page!

And maybe to have some online documentation, and to mention jekyll, or have a license notice, and maybe some screenshots, and a contributing page with instructions and contacts, i don't know!

Anyway you can just ask Gab he's the one who helped me out the most on the interface of the project. 

See you!

**Comment by bandang0** [comment]
But indeed I think I have misunderstood: if it is about design (appearance) of the site then we absolutely have the same thing, and the Blank Theme does look like a place to start to design your own custom theme!

**Comment by alarsyo** [comment]
Yeah, this was mostly about appearance, originally we planned on having our own "GameBoy style" theme but kinda forgot about it later on.

Awesome ! @naganori-san see what you can do then ! 😄

**Comment by bandang0** [comment]
Yes I get you it would be awesome to have your own retro Game Boy theme! I hope we can meet up after the summer with @naganori-san and think about it. 

Still, I think SharpBoy deserves that its documentation (the three reports and the specification) be linked to somewhere on the site! But that's another story.
